docs/statsig-warehouse-native/features/monitor-an-experiment.md

```diff
@@ -0,0 +1,50 @@
+
+### Monitoring experiment health
+To monitor the status of your experiment, 
+ - Navigate to **Experiments** in the left-hand navigation panel in the Statsig console
+ - Select the experiment you want to monitor 
+ - **Experiment Health Checks** show alerts for problems with the experiment setup.  Hover over the icon and click on a check for more details.
+   
+   ![Screen Shot 2023-11-09 at 11 11 07 AM](https://github.com/statsig-io/docs/assets/135062389/375aa506-5dbf-42e5-b238-280c907f499b)
+
+    - **Pulse metrics available**  monitors availability of Pulse results, which are expected on the day after the experiment starts.  
+    - **Exposures are balanced** checks that the number of units exposed in each group matches the expected allocation.  The Sample Ratio Mismatch (SRM) check is performed using a Chi-Squared test of independence. It is possible for experiments to temporarily show small imbalances simply due to the randomness of the user assignment.  These will often resolve on their own after a couple days, while true SRM imbalances that originate from systematic assignment/logging problems tend to persist over time.  The following thresholds are used for displaying SRM alerts:
+      - **p-value between 0.001 and 0.01**: Warning (yellow) for possible imbalance. The p-value is not low enough to confidently say that there is a real imbalance.  In this case the recommendation is to wait and check again the next day.
+      - **p-value < 0.01 and group size differs from expected size by less than 0.1% absolute**: Warning (yellow) indicating that an imbalance is possible, but the impact to the experiment is expected to be small.  This scenario typically occurs in large web experiments (1M+ users) where small variations in performance across groups can cause a small fraction of exposures to be dropped for certain groups more than others. 
+      - **p-value < 0.001 and more than 0.1% absolute deviation from expected group size**: Alert (red) meaning there is likely a problem with the experiment exposures and experiment results may not be trustworthy.
+    - **Crossover users** monitors the percentage of units which are exposed to more than one experiment group for a given experiment. Statsig discards crossover users from the experiment analysis, which may reduce your sample size and statistical power. The following thresholds are used for displaying crossover users alerts:
+      - **Crossover user percentage between 1% and 10%**: Warning (yellow) meaning that the presence of a number of crossover users has been detected, but the impact on the experiment is expected to be relatively small.
+      - **Crossover user percentage over 10%**: Alert (red) indicating that there are a substantial number of crossover users, which can indicate a problem with you experiment setup.
+    - **User metrics were computed** 
```

Comment:
This check confirms that we were able to join exposure data to metric data and produce experiment results.